consumer-test/runTests.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+set -ex
+
+LIBRARY_VERSION_UNDER_TEST=$(git log --format="%H" -n 1)
+REPOSITORY_UNDER_TEST=${TRAVIS_REPO_SLUG:-SAP/jenkins-library}
+
+rm -rf workspace
+git clone -b consumer-test https://github.com/sap/cloud-s4-sdk-book workspace
+cp -f jenkins.yml workspace
+cd workspace
+sed -i -e "s:__REPO_SLUG__:${REPOSITORY_UNDER_TEST}:g" jenkins.yml
+echo "@Library(\"piper-library-os@$LIBRARY_VERSION_UNDER_TEST\") _" | cat - Jenkinsfile > temp && mv temp Jenkinsfile
```

Comment:
Seems more complicated to me than required. Instead of file seding I would like to propose to configure the pipeline which should run on the level of the Jenkins config.

Reply:
The challenge here is to enforce that the PR version of piper os is running in our pipeline, and setting it in the Jenkins config would not do the trick because we set a version in our pipeline, which is overridden here.

Reply:
that is something which could be changed. Seding in Jenkinsfile files is IMO inferior compared to configuring it on the level of the Jenkins, especially if the Jenkins is launched for executing one single job. But maybe it is a matter of taste.

Reply:
What is your suggestion? I'm open for better suggestions, but I don't understand what your approach is. What I say is: Setting the version in `jenkins.yml` won't do it for us, because the version is overriden by the lib.